plugins/azure-devops-backend/README.md

```diff
@@ -112,6 +118,39 @@ The Azure DevOps backend plugin includes the `AzureDevOpsAnnotatorProcessor` whi
   }
 ```
 
+To use this with the New Backend System you'll want to create a [backend module extension for the Catalog](https://backstage.io/docs/backend-system/building-backends/migrating#other-catalog-extensions) if you haven't already. Here's a basic example of this assuming you are only adding the `AzureDevOpsAnnotatorProcessor`, this would go in your `packages/backend/index.ts`:
+
+```diff
+   import { createBackend } from '@backstage/backend-defaults';
++  import { catalogProcessingExtensionPoint } from '@backstage/plugin-catalog-node/alpha';
++  import { coreServices, createBackendModule } from '@backstage/backend-plugin-api';
++  import { AzureDevOpsAnnotatorProcessor } from '@backstage/plugin-azure-devops-backend';
+
++  const catalogModuleCustomExtensions = createBackendModule({
++    pluginId: 'catalog', // name of the plugin that the module is targeting
++    moduleId: 'custom-extensions',
++    register(env) {
++      env.registerInit({
++        deps: {
++          catalog: catalogProcessingExtensionPoint,
++          config: coreServices.rootConfig,
++        },
++        async init({ catalog, config }) {
++          catalog.addProcessor(AzureDevOpsAnnotatorProcessor.fromConfig(config));
++        },
++      });
++    },
++  });
```

Comment:
Can we simplify this?

1. Is there any harm in including `AzureDevOpsAnnotatorProcessor` by default in the core `azureDevOpsPlugin`?
2. If we don't include it by default, could we include this module as an additional module in the azure devops plugin, and then have the consumers import that.  That could simplify this installation down to:
```ts
import { AzureDevopsAnnotationModule } from '@backstage/plugin-azure-devops-backend/alpha';
backend.add(AzureDevopsAnnotationModule);
```

Reply:
I'd prefer to have this processor be opt-in. Not every one needs it, especially the vast majority of current Adopters of this plugin. But I for sure am open to making this easier to setup, I'll look into what you've suggested.

Reply:
No worries, I'll put together a follow up PR on this tomorrow 👍

Reply:
https://github.com/backstage/backstage/pull/21796 resolve this 🚀